### ISSUE TITLE
Cannot use local OpenAPI file, throws class cast exception

### ISSUE THREAD

**Issue by mimkorn** [EMResearch/EvoMaster#283]
Hi,

wanted to try out EvoMaster for our company's needs. I tried to provide it with an OpenAPI spec file like so --bbSwaggerUrl file:///Users/someone/Downloads/fjd/spa-templates.yaml, however, it only accepts files on the internet. Haven't found an option for accepting local files. Maybe something to consider adding. 

```
Caused by: java.lang.IllegalStateException: Failed to connect to file:///Users/someone/Downloads/fjd/spa-templates.yaml: sun.net.www.protocol.file.FileURLConnection cannot be cast to java.net.HttpURLConnection
	at org.evomaster.core.problem.rest.OpenApiAccess.connectToServer(OpenApiAccess.kt:52)
	at org.evomaster.core.problem.rest.OpenApiAccess.getOpenAPI(OpenApiAccess.kt:18)
	at org.evomaster.core.problem.rest.service.AbstractRestSampler.initForBlackBox(AbstractRestSampler.kt:141)
	at org.evomaster.core.problem.rest.service.AbstractRestSampler.initialize(AbstractRestSampler.kt:57)
	at com.netflix.governator.lifecycle.LifecycleMethods.methodInvoke(LifecycleMethods.java:313)
	... 26 common frames omitted
```

**Comment by arcuri82** [comment]
Hi,
  thanks. that is indeed an interesting feature, which should not be too difficult to implement

**Comment by arcuri82** [comment]
this is now implemented in the master branch.
if you want to try it out before the next release, you will need to build the `evomaster.jar` from source

**Comment by arcuri82** [comment]
this should now be fixed in the new 1.2.0 release. if not, please open a new issue
